test-data/unit/check-class-namedtuple.test

```diff
@@ -583,7 +583,7 @@ class MagicalFields(NamedTuple):
     x: int
     def __slots__(self) -> None: pass  # E: Cannot overwrite NamedTuple attribute "__slots__"
     def __new__(cls) -> None: pass  # E: Cannot overwrite NamedTuple attribute "__new__"
-    def _source(self) -> int: pass  # E: Cannot overwrite NamedTuple attribute "_source"
+    def _source(self) -> int: pass  # _source is ok
```

Comment:
I think this is an oversight in ``typing``. ``_source`` should probably be prohibited. I will make a PR to ``typing`` now.